server/user_account/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
+
+class CustomUser(AbstractBaseUser):
+    USERS = (
+        ('SA', 'System Admin'),
+        ('IM', 'Inventory Manager'),
+        ('SK', 'Stock Keeper'),
+    )
+    user_name = models.CharField(max_length=30, unique=True)
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    role = models.CharField(max_length=2, choices=USERS)
```

Comment:
Why max length of 2?

Reply:
Because I specify that each different user role is either SA, IM, SK respectively for system admin, inventory manager, and stock keeper.

Reply:
Makes sense.